docs/source/channel_update_tutorial.rst

```diff
@@ -476,25 +438,25 @@ Use the ``peer channel fetch`` command to retrieve this block:
 
 .. code:: bash
 
-  peer channel fetch 0 mychannel.block -o orderer.example.com:7050 -c $CHANNEL_NAME --tls --cafile $ORDERER_CA
+  peer channel fetch 0 channel1.block -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com -c channel1 --tls --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem
 
 Notice, that we are passing a ``0`` to indicate that we want the first block on
 the channel's ledger; the genesis block. If we simply passed the
 ``peer channel fetch config`` command, then we would have received block 3 -- the
 updated config with Org3 defined. However, we can't begin our ledger with a
 downstream block -- we must start with block 0.
 
-If successful, the command returned the genesis block to a file named ``mychannel.block``.
+If successful, the command returned the genesis block to a file named ``channel1.block``.
 We can now use this block to join the peer to the channel. Issue the
 ``peer channel join`` command and pass in the genesis block to join the Org3
 peer to the channel:
 
 .. code:: bash
 
-  peer channel join -b mychannel.block
+  peer channel join -b channel1.block
 
 To join by a snapshot, follow the instruction in `Taking a snapshot <peer_ledger_snapshot.html#taking-a-snapshot>`__
-to take a snapshot on an existing peer. The snapshot should be taken after Org3 has been added to ``mychannel``
+to take a snapshot on an existing peer. The snapshot should be taken after Org3 has been added to ``channel1``
 to ensure that the snapshot contains the updated channel configuration including Org3.
 Locate the snapshot directory, copy it to the filesystem of the new Org3 peer, and issue the
 ``peer channel joinbysnapshot`` command using the path to the snapshot on your file system.
```

Comment:
The below section is not in the scope of this PR, but given that we have changed the recommended settings to get blocks from the orderer, it may best to set only one option and link to the deployment guide, or maybe move most of this content there

Reply:
can do this in another PR